cmd/backup-manager/app/backup/manager.go

```diff
@@ -252,6 +252,9 @@ func (bm *Manager) performBackup(backup *v1alpha1.Backup, db *sql.DB) error {
 	if oldTikvGCTimeDuration < tikvGCTimeDuration {
 		err = bm.SetTikvGCLifeTime(db, oldTikvGCTime)
 		if err != nil {
+			if backupErr != nil {
```

Comment:
Please also update restore/manager.go

Reply:
updated in https://github.com/pingcap/tidb-operator/pull/2926/commits/eaf73927bbd9f066362febba1db3b582f87d0444